Sources/ChatUI/ChatInChannel/MessageField/Location/View/LocationSelector.swift

```diff
@@ -72,7 +72,9 @@ public struct LocationSelector: View {
                 // Dimiss button & Search bar
                 HStack {
                     Button(action: dismiss) {
+
```

Comment:
I found some unnecessary vertical white spaces.  Please allow me to modify by granting the writing permission